docs/source/routing/license.mdx

```diff
@@ -0,0 +1,153 @@
+
+</Note>
+
+An offline license is valid for the lesser of the duration of your contract with Apollo, or one year, with an added grace period of 28 days. You are responsible for keeping your offline license files up to date within your infrastructure by rerunning `rover license fetch` to fetch updated license files.
+
+### Set up offline license for the GraphOS Router
+
+Follow these steps to configure an GraphOS Router to use an offline license:
+
+1. Fetch an offline license by running [`rover license fetch`](/rover/commands/license/#license-fetch) with the ID of the graph from which you want to fetch a license:
+
+    ```bash
+    rover license fetch --graph-id <apollo-graph-id>
+    ```
+
+1. Provide the offline license to your router on startup. The router accepts an offline license in a few ways:
+
+    1. [`--license <license_path>`](/graphos/reference/router/configuration#--license) CLI option, with an argument containing an absolute or relative path to an offline license file
+    1. [`APOLLO_ROUTER_LICENSE_PATH`](/graphos/reference/router/configuration#--license) environment variable, containing an absolute or relative path to an offline license file 
+    1. [`APOLLO_ROUTER_LICENSE`](/graphos/reference/router/configuration#--license) environment variable, containing the stringified contents of an offline license file
+
+    <Note>
+
+    - The router checks the CLI option and environment variables in the listed order, then it uses the value of the first option or variable that is set.
+    - The `--license <license_path>` option is only available when running the router binary. When running a router with `rover dev`, you must use environment variables to provide your offline license.
+    
+    </Note>
+
+1. Configure the router to use a local supergraph schema by setting one of the following:
+
+    * [`--s/-supergraph`](/graphos/reference/router/configuration#-s----supergraph) CLI option, with an argument containing an absolute or relative path to supergraph schema file
+    * [`APOLLO_SUPERGRAPH_PATH`](/graphos/reference/router/configuration#-s----supergraph) environment variable, containing an absolute or relative path to supergraph schema file
+    * [`APOLLO_SUPERGRAPH_URLS`](/graphos/reference/router/configuration#-s----supergraph) environment variable, containing URLs to supergraph schemas
+
+1. (**Recommended**) Configure the router to report usage metrics to GraphOS in a best-effort basis by setting both the [`APOLLO_KEY`](/graphos/reference/router/configuration#apollo_key) and [`APOLLO_GRAPH_REF`](/graphos/reference/router/configuration#apollo_graph_ref) environment variables. 
+
+    These metrics are necessary for several important GraphOS features (operations checks, field insights, operation traces, contracts). Sending them best-effort incurs no performance or uptime penalties. 
+
+## Licenses with local development
```

Comment:
More users need to know about licenses for local development. Consider moving this section above offline licenses.